slash/core/session.py

```diff
@@ -94,6 +95,8 @@ def deactivate(self):
 
     @contextmanager
     def get_started_context(self):
+        self.host_fqdn = socket.getfqdn()
```

Comment:
@ayalash cache this value somewhere. When running tests with VPN enabled on a mac this can cause a 1000% slowdown